recipe/meta.yaml

```diff
@@ -1,13 +1,13 @@
 package:
     name: flopy
-    version: 3.2.4
+    version: 3.2.5
 
 source:
     git_url: https://github.com/modflowpy/flopy.git
-    git_tag: 3.2.4
+    git_tag: 3.2.5
 
 build:
-    number: 0
+    number: 1
```

Comment:
You don't need to bump the build number when updating to a new release. We only update the build number when we modify the same release.